ziti/ziti.go

```diff
@@ -447,6 +447,9 @@ func (context *contextImpl) GetServiceId(name string) (string, bool, error) {
 }
 
 func (context *contextImpl) GetService(name string) (*edge.Service, bool) {
+	if err := context.initialize(); err != nil {
```

Comment:
if you call GetService without this change you'll be surprised when you get no results.